controller/v1/system.py

```
@@ -7,10 +11,38 @@
+    """운영팀 전체 정보 및 리스트 조회"""
+    return await team_service.get_team_members(db)
+
+
+@router.get("/team/{member_id}", response_model=TeamMemberDetailResponse)
+async def get_team_member_detail(member_id: int, db: AsyncSession = Depends(get_db)):
+    """특정 팀원의 상세 한마디 조회"""
+    return await team_service.get_team_member_detail(db, member_id)
+
+
+@router.get("/admin-character", response_model=dict)
+async def get_admin_character(db: AsyncSession = Depends(get_db)):
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)